extensions/amp-story/1.0/amp-story-quiz.css

```diff
@@ -0,0 +1,145 @@
+    color: #5F6368 !important;
+    border: solid 1px !important;
+    border-color: #DADCE0 !important;
+    /* TODO: Why won't this notation work? */
+    /* font: 16px/20px !important; */
+    font-size: 16px !important;
+    line-height: 20px !important;
+}
+
+.i-amp-story-quiz-option-post-selection.i-amp-story-quiz-option-selected {
+    border: none !important;
+    color: white !important;
+    padding-top: 9px !important;
+    padding-bottom: 9px !important;
+}
+
+.i-amp-story-quiz-answer-choice {
+    display: grid !important;
+    justify-items: center !important;
+    align-items: center !important;
+    height: 24px !important;
+    width: 24px !important;
+    border-radius: 50% !important;
+    border-style: solid !important;
+    border-width: 2px !important;
+    padding: 5px !important;
+    color: var(--accent-color) !important;
+    font-weight: 800 !important;
+    font-size: 16px !important;
+    line-height: 16px !important;
+}
+
+.i-amp-story-quiz-option-post-selection:not([correct]) > .i-amp-story-quiz-answer-choice {
+    color: var(--incorrect-color) !important;
+}
+
+.i-amp-story-quiz-option-post-selection[correct] > .i-amp-story-quiz-answer-choice {
+    color: var(--correct-color) !important;
+}
+
+.i-amp-story-quiz-option-post-selection.i-amp-story-quiz-option-selected:not([correct]) {
+    background-color: var(--incorrect-color) !important;
+}
+
+.i-amp-story-quiz-option-post-selection.i-amp-story-quiz-option-selected[correct] {
+    background-color: var(--correct-color) !important;
+}
+
+.i-amp-story-quiz-option-post-selection.i-amp-story-quiz-option-selected > .i-amp-story-quiz-answer-choice {
+    border: none !important;
+    padding: 7px !important;
+}
+
+/** TODO(jackbsteinberg): is font-size: 0; a11y friendly? */
+
+.i-amp-story-quiz-option-post-selection[correct] > .i-amp-story-quiz-answer-choice {
+    background-color: white !important;
+    background-image: url('data:image/svg+xml;charset=utf-8,<svg xmlns="http://www.w3.org/2000/svg" width="24px" height="24px" viewBox="0 0 24 24" fill="#5BBA74"><path fill="none" d="M0 0h24v24H0z"/><path d="M9 16.2L4.8 12l-1.4 1.4L9 19 21 7l-1.4-1.4L9 16.2z"/></svg>') !important;
+    background-repeat: no-repeat !important;
+    background-position: center !important;
+    font-size: 0 !important;
+}
+
+.i-amp-story-quiz-option-post-selection:not([correct]) > .i-amp-story-quiz-answer-choice {
+    background-color: white !important;
+    background-image: url('data:image/svg+xml;charset=utf-8,<svg xmlns="http://www.w3.org/2000/svg" width="24px" height="24px" viewBox="0 0 24 24" fill="#F34E4E"><path d="M19 6.41L17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12z"/><path d="M0 0h24v24H0z" fill="none"/></svg>') !important;
+    background-repeat: no-repeat !important;
+    background-position: center !important;
+    font-size: 0 !important;
+}
+
+.i-amp-story-quiz-percentage {
+    padding-left: 10px !important;
```

Comment:
Please avoid padding-right/left specific rules as much as possible since they have to be overridden for RTL languages. Usually you can find workarounds with the display flex/grid that you picked.

Reply:
Changed where I used this and added RTL support